vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -489,6 +488,14 @@ def forward_cuda(
                 global_num_experts=global_num_experts,
                 expert_map=expert_map,
             )
+            if self.has_bias and not isinstance(self.fused_experts,
+                                                FusedMoEModularKernel):
+                kwargs.update({
+                    "w1_bias": getattr(layer, "w13_bias", None),
+                    "w2_bias": getattr(layer, "w2_bias", None),
+                })
```

Comment:
This is a silent error. It seems we should raise an exception if `self.has_bias and isinstance(self.fused_experts, FusedMoEModularKernel)`